.github/data/dependencies.yml

```diff
@@ -41,13 +41,17 @@
     - id: io.buildpacks.stacks.bionic
       mixins:
         - libexpat1
+    - id: io.buildpacks.stacks.jammy
+      mixins:
+        - libexpat1
```

Comment:
I think `libexpat1` is already in the stacks now so we don't need it as a mixin anymore